src/errorpage.cc

```diff
@@ -758,6 +758,15 @@ ErrorState::Convert(char token, bool building_deny_info_url, bool allowRecursion
             p = "-";
         break;
 
+    case 'A': {
+        const Ip::Address *addr;
+        if (request && (addr = FindListeningPortAddress(request.getRaw())))
```

Comment:
But if you see the code above and below there are cases where checks are being made:

if (request) {
...request.getRaw()...
}

But since I have no idea about smart pointer object, I will take your word and change the code accordingly. i.e. wont check if request is NULL or not.

Thank you